src/mame/f256/tiny_vicky.h

```diff
@@ -0,0 +1,83 @@
+// license:BSD-3-Clause
+// copyright-holders:Daniel Tremblay
+#ifndef MAME_F256_TINY_VICKY_H
+#define MAME_F256_TINY_VICKY_H
+
+#define CHAR_HEIGHT     8
+#define CHAR_WIDTH      8
+
+//#include <atomic>
+
+class tiny_vicky_video_device : public device_t
+{
+public:
+    tiny_vicky_video_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+    tiny_vicky_video_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock = 0);
+    //tiny_vicky_video_device() = default;
+
+    // screen update
+    uint32_t screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
+    void stop()
+    {
+        m_running = false;
+        m_videoram_ptr = nullptr;
+        m_iopage0_ptr = nullptr;
+        m_iopage1_ptr = nullptr;
+        m_iopage2_ptr = nullptr;
+        m_iopage3_ptr = nullptr;
+    }
```

Comment:
This will create confusion with `device_t::stop()`.  You should be using the regular device lifecycle functions.